front/app/containers/ProjectsShowPage/SucessModal.tsx

```diff
@@ -29,6 +30,10 @@ const SuccessModal = ({ projectId }: Props) => {
   const [queryParams] = useSearchParams();
   const showModalParam = !!queryParams.get('show_modal');
   const phaseIdParam = queryParams.get('phase_id');
+  const [newIdeaIdParam] = useState(
+    queryParams.get('new_idea_id') ?? undefined
```

Comment:
Added this new query param to get the idea id of the just submitted survey on the project page, where we trigger the modal